src/status_im/ui/screens/chat/components/reply.cljs

```diff
@@ -144,7 +144,7 @@
   (when-not (= reply @had-reply)
     (reset! had-reply reply)
     (when reply
-      (js/setTimeout #(input-focus text-input-ref) 250))))
+      (js/setTimeout #(input-focus text-input-ref) 0))))
```

Comment:
is the time out necessary? 🤔

Reply:
Thanks @J-Son89 for your reviews.

Yes `setTimeout` is necessary. `setTimeout(() => statement, 0)` is not the same as `statement`. Having a `setTimeout` of 0 ensures that the statement is enqueued and will get executed ASAP. Previously it was 250ms which I think is not necessary as we want to focus on the input right away.

Reply:
cool, assumed as much but just wanted to check. Do you think we should add a comment for this? 🤔